examples/shortest.config.ts

```diff
@@ -0,0 +1,12 @@
+import type { ShortestConfig } from "@antiwork/shortest";
```

Comment:
Do we need this, would prefer just having the main config apply to this.

e.g.

pnpm shortest examples

Reply:
^ would be good to add to the main README too